scripts/mod2_pcoa.py

```diff
@@ -37,6 +50,12 @@ def body_site(coords, mapping_file):
                      index_col='#SampleID')
     mf = mf.loc[o.site_ids]
 
+    if samples is None:
+        samples = mf.index
+    else:
+        samples = set(samples.split(',')).intersection(set(o.site_ids))
```

Comment:
I thought mf is the mapping file and o is the coordinates file. This may be interpretation issue, but if this is reading from o, you could theoretically have a subset of samples from the mapping file in o, but pass in the full mapping file sample list?